Regulator_Switching.dcm

```diff
@@ -3474,26 +3474,45 @@ K Murata isolated isolation dc-dc converter
 F http://power.murata.com/data/power/ncl/kdc_nma.pdf
 $ENDCMP
 #
+$CMP NXE1S0303MC
+D 1W 3.3V to 3.3V 303mA DC-DC Converter with 3kV isolation
+K isolated isolation dc-dc converter transformer
+F http://power.murata.com/data/power/ncl/kdc_nxe1.pdf
+$ENDCMP
+#
+$CMP NXE1S0305MC
+D 1W 3.3V to 5V 200mA DC-DC Converter with 3kV isolation
+K isolated isolation dc-dc converter transformer
+F http://power.murata.com/data/power/ncl/kdc_nxe1.pdf
+$ENDCMP
+#
+$CMP NXE1S0505MC
+D 1W 5V to 5V 200mA DC-DC Converter with 3kV isolation
+
```

Comment:
There is a weird empty line here

Reply:
weird indeed, I made all changes through the KiCAD symbol editor, not edited the files by hand.
Anyhow, I removed the empty line.
After checking in the symbol editor, I found an odd character at the end of the description line.
Should be fine, now